package.json

```diff
@@ -24,7 +24,8 @@
     "debug": "^2.2.0",
     "ioredis": "^2.2.0",
     "loopback-connector": "^2.4.0",
-    "msgpack5": "^3.4.0"
+    "msgpack5": "^3.4.0",
+    "strong-globalize": "^2.6.21"
```

Comment:
I think you're installing from staging. Should be `^2.6.5` ATM -- `^2.6.6` is better and fixes the bug I had during demo but I'm not sure if they released it yet.